src/CheckoutSdk/Disputes/GetDisputesRequest.cs

```diff
@@ -0,0 +1,122 @@
+            int? limit = null,
+            int? skip = null,
+            DateTimeOffset? from = null,
+            DateTimeOffset? to = null,
+            string id = null,
+            string statuses = null,
+            string paymentId = null,
+            string paymentReference = null,
+            string paymentArn = null,
+            bool? thisChannelOnly = null
+            )
+        {
+            Limit = limit;
+            Skip = skip;
+            From = from;
+            To = to;
+            Id = id;
+            Statuses = statuses;
+            PaymentId = paymentId;
+            PaymentReference = paymentReference;
+            PaymentArn = paymentArn;
+            ThisChannelOnly = thisChannelOnly;
+        }
+
+        /// <summary>
+        /// Gets or sets the numbers of results to return. 1 .. 250, default is 50.
+        /// </summary>
+        public int? Limit { get; set; }
+
+        /// <summary>
+        /// Gets or sets the number of results to skip, default is 0.
+        /// </summary>
+        public int? Skip { get; set; }
+
+        /// <summary>
+        /// Gets or sets the ISO-8601 date and time from which to filter disputes, based on the dispute's last_update field.
+        /// </summary>
+        public DateTimeOffset? From { get; set; }
+
+        /// <summary>
+        /// Gets or sets the ISO-8601 date and time until which to filter disputes, based on the dispute's last_update field.
+        /// </summary>
+        public DateTimeOffset? To { get; set; }
+
+        /// <summary>
+        /// Gets or sets the unique identifier of the dispute.
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets one or more comma-separated statuses.
+        /// This works like a logical OR operator.
+        /// </summary>
+        public string Statuses { get; set; }
+
+        /// <summary>
+        /// Gets or sets the unique identifier of the payment.
+        /// </summary>
+        public string PaymentId { get; set; }
+
+        /// <summary>
+        /// Gets or sets an optional reference (such as an order ID) that you can use later to identify the payment.
+        /// </summary>
+        public string PaymentReference { get; set; }
+
+        /// <summary>
+        /// Gets or sets the acquirer reference number (ARN) that you can use to query the issuing bank.
+        /// </summary>
+        public string PaymentArn { get; set; }
+
+        /// <summary>
+        /// Gets or sets a boolean that, if true, only returns disputes of the specific channel that the secret key is associated with.
+        /// Otherwise, returns all disputes for that business.
+        /// </summary>
+        public bool? ThisChannelOnly { get; set; }
+
+        /// <summary>
+        /// Gets the query path of the GetDisputesRequest.
+        /// </summary>
+        public string PathWithQuery(string path) {
+
+            var query = new StringBuilder("");
```

Comment:
```suggestion
```
Remove unused `StringBuilder` (note: the `QueryHelpers.AddQueryString` uses a string builder to format the parameters.

Reply:
removed